test/sort-objects.test.ts

```diff
@@ -478,251 +478,651 @@ describe(ruleName, () => {
       },
     )
 
-    ruleTester.run(
-      `${ruleName}(${type}): works with complex dependencies`,
-      rule,
-      {
-        valid: [],
-        invalid: [
-          {
-            code: dedent`
-              let Func = ({
-                a,
-                b = a + c + d,
-                c,
-                d,
-              }) => {
-                // ...
-              }
-            `,
-            output: dedent`
-              let Func = ({
-                a,
-                c,
-                d,
-                b = a + c + d,
-              }) => {
-                // ...
-              }
-            `,
-            options: [options],
-            errors: [
-              {
-                messageId: 'unexpectedObjectsOrder',
-                data: {
-                  left: 'b',
-                  right: 'c',
+    describe('detects dependencies', () => {
+      ruleTester.run(
+        `${ruleName}(${type}): works with complex dependencies`,
+        rule,
+        {
+          valid: [],
+          invalid: [
+            {
+              code: dedent`
+                let Func = ({
+                  a,
+                  b = a + c + d,
+                  c,
+                  d,
+                }) => {
+                  // ...
+                }
+              `,
+              output: dedent`
+                let Func = ({
+                  a,
+                  c,
+                  d,
+                  b = a + c + d,
+                }) => {
+                  // ...
+                }
+              `,
+              options: [options],
+              errors: [
+                {
+                  messageId: 'unexpectedObjectsOrder',
+                  data: {
+                    left: 'b',
+                    right: 'c',
+                  },
                 },
-              },
-            ],
-          },
-          {
```

Comment:
Test removed because function bodies no longer get taken into account for dependencies. See `ignores function body dependencies` test.